website/docs/core-concepts/data-access-and-binding/querying-a-database/README.md

```diff
@@ -81,13 +81,23 @@ All query results are stored in the [data](/reference/appsmith-framework/query-o
 
 All **API Queries** return the exact API response of the endpoint.
 
-## Using Queries in applications
+## Using queries in applications
 
 Once you have successfully run a Query, you can use it in your application to
 
 * [Display Data](/core-concepts/data-access-and-binding/displaying-data-read)
 * [Capture Data](/core-concepts/data-access-and-binding/capturing-data-write)
 
+## Debugging queries
```

Comment:
sorry this should have been 'Debugging errors' and not queries. . Just that small change